views/amp-access/article.html

```diff
@@ -79,7 +83,7 @@
     <main role="main">
     <article>
 
-      <section id="meter-notif" amp-access="access" class="meter-section" data-z-depth="2">
+      <section id="meter-notif" amp-access="access OR error" class="meter-section" data-z-depth="2">
```

Comment:
The purpose is just to demo how one can use the fallback do show or hide a section if an error occurs. I don't think we have a recommendation around this at the moment.